src/libs/Navigation/AppNavigator/Navigators/OnboardingModalNavigator/index.tsx

```diff
@@ -17,7 +18,7 @@ import * as Report from '@userActions/Report';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import SCREENS from '@src/SCREENS';
-import Overlay from './Overlay';
```

Comment:
What is the need for this change?

Reply:
This is just a code clean up... If you don't agree, I will revert. just let me know.

Reply:
Yup. Let's revert it.

Reply:
![image](https://github.com/Expensify/App/assets/25519849/f81c033d-5e48-4d92-8f2f-b63e8ada184f)
IT IS A ESLINT RULE NOW :)

Reply:
I think this is related to eslint linked with your vsc not the app.

Reply:
![image](https://github.com/Expensify/App/assets/25519849/bb9140fa-017b-48aa-ad98-78ff5e8cd79f)
This is not my IDE rule...EXPENSIFY RULE!!!